odkx-src/sync-endpoint-manual-setup.rst

```diff
@@ -197,3 +199,42 @@ Stopping ODK-X Sync Endpoint
 
       $ docker volume rm (docker volume ls -f "label=com.docker.stack.namespace=syncldap" -q)
 
+.. _sync-anonymous:
+
+Anonymous Access for ODK-X Sync Endpoint
+-----------------------------------------
+
+Checking for Anonymous User Access
+  If you have already created the Docker Config and deployed the Docker Stack.
+  You can navigate to http://[IP_ADDRESS]/web-ui/admin/users
+  or http://[IP_ADDRESS]/odktables/[APP_NAME]/usersInfo and look for *user_id: anonymous*,
```

Comment:
`user_id: anonymous` does not imply anonymous access. You need to check if `anonymous` is a member of the group `ROLE_USER`. And membership in `ROLE_SYNCHRONIZE_TABLES` to sync data.